python/graphframes/examples/belief_propagation.py

```diff
@@ -19,7 +19,6 @@
 
 from pyspark import SparkConf, SparkContext
 from pyspark.sql import SQLContext, functions as sqlfunctions, types
-from pyspark.tests import QuietTest as SuppressSparkLogs
```

Comment:
The class `QuietTest` is annoying. In spark 2.4 it is in `pyspark.tests` package but in spark 3.0 it is in pyspark.testing` package.
So I remove it from this file. The example code here should not use some class inside pyspark test.